### ISSUE TITLE
Date.UTC not supported, possible solution

### ISSUE THREAD

**Issue by jondkinney** [schickling/timemachine#1]
Hi there,

I was trying to use this, but I use Date.UTC in a few places (coffeescript)

```
@dateSpecifiedDaysFromToday = (days) ->
  today = new Date()
  today_in_ms = Date.UTC(today.getFullYear(), today.getMonth(), today.getDate())
  one_day_in_ms = 1000 * 60 * 60 * 24
  tz_ms = today.getTimezoneOffset() * 60 * 1000
  new Date(today_in_ms + (days * one_day_in_ms) + tz_ms)
```

So I had to add something like this...

```
Date.UTC = function () {
  var date = new OriginalDate(self.dateString);
  var dateMs = date.getTime();
  var tzMs = date.getTimezoneOffset() * 60 * 1000;
  var utcDate = dateMs + tzMs
  return utcDate;
};
```

Is this correct? I'm not a JS timezone expert, so I'm not sure if getTimezoneOffset() will always return a number I can add to getTime() to move the clock the right direction. For instance... I'm in CST so it's GMT - 6 so I need to add the timezone offset to get back to GMT. But if you're GMT + 6 would JS return a negative getTimezoneOffset() value? I _think/hope_ it would...

Thanks much!

**Comment by lucasjans** [comment]
@jondkinney  Was there ever a pull request on this?

**Comment by jondkinney** [comment]
@lucasjans I didn't get to it, no. I wasn't sure how to add the test case.

**Comment by gajus** [comment]
@schickling Thats still an open issue. This makes it not work with `moment` package.

**Comment by schickling** [comment]
Hi @gajus. I'm sorry I don't put much efforts in the development of this package at the moment. But I'd be happy to merge a PR for this.

**Comment by xkjyeah** [comment]
@schickling  Hi, I see you have fixed this in master. Could you publish the latest version to npm?

**Comment by schickling** [comment]
Done. Just published `0.2.8`.
